buku

```diff
@@ -725,7 +728,9 @@ class BukuDb:
             url_redirect: bool = False,
             tag_redirect: bool | str = False,
             tag_error: bool | str = False,
-            del_error: Optional[Set[int] | range] = None) -> int:  # Optional[IntSet]
+            del_error: Optional[Set[int] | range] = None,
+            tags_fetch: bool = True,
+            tags_except: Optional[str] = None) -> int:  # Optional[IntSet]
```

Comment:
New parameters for `add_rec()`

The method has no `*` enforcing keyword arguments, so I added them to the end of the list